modules/apps/friendly-url/friendly-url-service/src/main/java/com/liferay/friendly/url/internal/servlet/FriendlyURLServlet.java

```diff
@@ -108,20 +106,6 @@ public Redirect getRedirect(
 		Group group = groupLocalService.fetchFriendlyURLGroup(
 			companyId, friendlyURL);
 
-		if (group == null) {
-			String screenName = friendlyURL.substring(1);
-
-			User user = userLocalService.fetchUserByScreenName(
-				companyId, screenName);
-
-			if (user != null) {
-				group = user.getGroup();
-			}
-			else if (_log.isWarnEnabled()) {
-				_log.warn("No user exists with friendly URL " + screenName);
-			}
-		}
-
```

Comment:
@holatuwol @hhtank: why is this fallback removed? This allows the users to look up other users' personal sites if they don't know the actual friendly URLs. Is there any reason we want to disable this feature?

Reply:
I removed it because I thought it only existed because of the code in `GroupLocalService`. If that's not the case, you can just use interactive rebase to drop the commit I added instead of reverting it.

Reply:
Got it, thanks @holatuwol. The scenario you described is also the same issue when a user is created with a screen name that matches a site's friendly url. In that case, the user's personal site cannot be accessed with the screen name but only with the actual friendly url.

We need to think through a better way to handle user's screen name and personal site friendly URLs. For the time being, we decided to keep this feature to avoid causing any breaking changes.